onnxruntime/core/session/inference_session.cc

```diff
@@ -419,6 +422,25 @@ common::Status InferenceSession::TransformGraph(onnxruntime::Graph& graph,
         oss << "(" << node.Op()->since_version() << ")";
       }
       return Status(common::ONNXRUNTIME, common::NOT_IMPLEMENTED, oss.str());
+    } else {
+      if (is_verbose_mode) {  // TODO: should we disable this if the number of nodes are above a certain threshold?
+        std::string node_str = node.OpType();
```

Comment:
We could dump this information and the nodename to id mapping into a graphviz DAG file. It would be easier to read.

Reply:
Sure, we can do this as an enhancement to this later. For now, just wanted to provide a quick and easy way to dump and read this info. Will help us avoid a lot of support issues.